packages/web/components/related-assets/related-assets.tsx

```diff
@@ -0,0 +1,53 @@
+import Image from "next/image";
+
+import caretDown from "../../public/icons/caret-down.svg";
+
+export default function RelatedAssets() {
+  return (
+    <div className="flex flex-col gap-8 rounded-5xl border-[1px] border-osmoverse-800 bg-osmoverse-900 p-10">
```

Comment:
Change hardcoded "border-[1px]" with "border", by default it's 1px on tailwind styleguide.